### ISSUE TITLE
Rendering issue when viewing from tablet

### ISSUE THREAD

**Issue by b9Joker108** [tldr-pages/tldr-python-client#229]
Hi

Thank you so much for this project!

I installed the project on my Android tablet with `pip` in Termux.

It doesn't seem to get rendering correctly, please see screen capture uploaded. 
![Screenshot_20240212_125040_Termux](https://github.com/tldr-pages/tldr/assets/147242971/7f02749b-d33a-4e36-b82b-34d9b8967cc5)

**Comment by kbdharun** [comment]
Hi, thanks for the issue.

I tested the same (Termux on Android) with Landscape mode on a Samsung mobile and it rendered fine for me, can you check if your tablet zooms Termux window automatically?

Alternatively, can you try installing our [rust client](https://github.com/tldr-pages/tlrc) from Cargo and see if it renders fine? (i.e. first install Cargo with `pkg install rust` then `cargo install tlrc`)

**Comment by b9Joker108** [comment]
Hi, thank you so much for your response. Prior to reading your response, I actually uninstalled the installation with pip, and tried to install it other ways, but had a stumbling lock with the version of node.js in Termux repositories. The Termux repository just upgrade to a new version of node.js, and now tlrc worked flawlessly and rendered correctly. I am still thankful for your response, and as I have got the package working, this case may be closed.

Cheers
Beauford
